homeassistant/components/kaleidescape/__init__.py

```diff
@@ -0,0 +1,86 @@
+"""The Kaleidescape integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING, NamedTuple
+
+from kaleidescape import Device as KaleidescapeDevice, KaleidescapeError
+
+from homeassistant.const import CONF_HOST, EVENT_HOMEASSISTANT_STOP, Platform
+from homeassistant.exceptions import ConfigEntryNotReady, HomeAssistantError
+
+from .const import DOMAIN
+
+if TYPE_CHECKING:
+    from homeassistant.config_entries import ConfigEntry
+    from homeassistant.core import Event, HomeAssistant
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = [Platform.MEDIA_PLAYER, Platform.REMOTE]
+
+
+class DeviceInfo(NamedTuple):
```

Comment:
We have a new enough python requirement that you could use a dataclass here instead

Reply:
Fixed.